modules/examples/routeguide/server/src/main/scala/io/ServerIO.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017-2018 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package example.routeguide.server.io
+
+import cats.effect.IO
+import freestyle.rpc.server._
+import freestyle.rpc.server.config.BuildServerFromConfig
+import freestyle.rpc.server.{AddService, GrpcConfig, ServerW}
+import freestyle.tagless.config.implicits._
+import example.routeguide.server.handlers.RouteGuideServiceHandler
+import example.routeguide.protocol.Protocols.RouteGuideService
+import example.routeguide.runtime._
+
+object ServerIO {
```

Comment:
The nested trait and object can be pulled up from this object, so we could remove this object.